src/main/webapp/maps_searchbox.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * This module's {@code initAutocomplete} sets up a search-box with auto-suggestions feature,
+ * so that the user can search for a specific place.
+ *
+ * When a search query is submitted,
+ * the map is centered at the requested place and a marker is displayed for it.
+ *
+ * By default, the Autocomplete widget provided by Places API
+ * supports only submitting queries for one of the suggetsions.
+ * This makes the searching process quite uncomportable, since the user can't press enter
+ * if they didn't write the exact name of the place, they had a typo, etc.
+ * To avoid this issue, I extended the autocomplete widget's eventlistener
```

Comment:
Avoid "I" in the documentation. We could write "To avoid the issue, the autocomplete widget's eventlistener has been extended".

Reply:
Removed.